lib/ibm/cloud/sdk/vpc/instances.rb

```diff
@@ -38,21 +38,115 @@ def params(start: nil, limit: nil, name: nil, vpc_id: nil, vpc_crn: nil, vpc_nam
 
         # Work with a single instance.
         class Instance < VPCInstance
+          TRANSITIONAL_STATES = %(pausing pending restarting resuming starting stopping)
```

Comment:
I think you intended for this to be an array of strings:

```
>> %(pausing pending restarting resuming starting stopping)
=> "pausing pending restarting resuming starting stopping"
>> %w[pausing pending restarting resuming starting stopping]
=> ["pausing", "pending", "restarting", "resuming", "starting", "stopping"]
```

What you have here has the problem of:
```
>> %(pausing pending restarting resuming starting stopping).include?("pausing pending")
=> true
```